src/commonMain/kotlin/exchange.dydx.abacus/state/manager/RpcInfo.kt

```diff
@@ -0,0 +1,15 @@
+package exchange.dydx.abacus.state.manager
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class RpcInfo(
+    @SerialName("rpc") val rpcUrl: String,
+    val name: String,
+)
+
+
+object RpcConfigs {
+    var chainIdToRpcMap: Map<String, RpcInfo> = mapOf()
```

Comment:
think we need a default map here in case the rpc request fails. which RPC endpoints do we actually need? the generated json in the web repo has so many chains (is this like every known EVM chain?).

Reply:
> is this like every known EVM chain?

yes, based on https://github.com/wevm/viem/tree/main/src/chains/definitions, see [thread](https://dydx-team.slack.com/archives/C042V24U87Q/p1720721749832189) if you are curious

Reply:
i don't think we need anything super complicated here. just defaulting to like the top few popular evm chains should be fine. ETH, OP, Arbitrum, Polygon, Avalanche, Base

Reply:
what is the benefit of a few vs all of them? Other than a slightly smaller json.

Reply:
don't have to do a file read

Reply:
fwiw, i wrote a [script](https://github.com/dydxprotocol/v4-web/pull/798) to update the rpc list from that repo, so it's no more/less work to maintain a more comprehensive set of chains